extensions/Worker.Extensions.Tables/src/Config/TablesBindingOptions.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Azure.Core;
+using Azure.Data.Tables;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.Tables.Config
+{
+    internal class TablesBindingOptions
+    {
+        public string? ConnectionString { get; set; }
+
+        public Uri? ServiceUri { get; set; }
+
+        public TokenCredential? Credential { get; set; }
+
+        public TableClientOptions? TableClientOptions { get; set; }
+
+        private TableServiceClient? TableServiceClient;
+
+        internal virtual TableServiceClient CreateClient()
+        {
+            if (ConnectionString is null && ServiceUri is null)
+            {
+                throw new ArgumentNullException(nameof(ConnectionString) + " " + nameof(ServiceUri));
```

Comment:
Incorrect values being passed to the ANE ctor - should be a param name not an error message